css/_container.scss

```
@@ -0,0 +1,283 @@
+@import url("https://fonts.googleapis.com/css2?family=Chivo:wght@300;700&display=swap");
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: $backgroundColor;
+  font-family: "Chivo", sans-serif;
+  height: 100vh;
+}
+
+main::before {
+  content: "";
+  background-image: url("/assets/mobile/image-host.jpg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+  opacity: 0.1;
+  z-index: -1;
+}
+main {
```

```suggestion
main {
    &::before {
      content: "";
      background-image: url("/assets/mobile/image-host.jpg");
      background-size: contain;
      background-repeat: no-repeat;
      position: absolute;
      top: 0px;
      right: 0px;
      bottom: 0px;
      left: 0px;
      opacity: 0.1;
      z-index: -1;
    }
```
Primenjuj nestovanje!